frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts

```diff
@@ -49,6 +49,21 @@ export const multusNAD = {
 };
 deepFreeze(multusNAD);
 
+export const sriovNAD = {
+  apiVersion: 'k8s.cni.cncf.io/v1',
+  kind: 'NetworkAttachmentDefinition',
+  metadata: {
+    name: `sriov-${testName}`,
+    namespace: testName,
+    labels: { automatedTest: testName },
+  },
+  spec: {
+    // config: '{ "cniVersion": "0.3.1", "type": "cnv-bridge", "bridge": "testbridge", "ipam": {} }',
```

Comment:
if its wrong you don't need to leave it as a comment, just remove it

This also a comment for the rest of the places you left the old code as a comment - if it's wrong: delete it, if it's not wrong then don't comment it out :-)